Lazy/Tests/MultiThreadedTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿// <copyright file="ILazy.cs" company="Pakhnusheva Alena">
+// Copyright (c) Pakhnusheva Alena. All rights reserved.
+// </copyright>
+using System;
+using ILazy;
+using LazyCalculation;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Tests
+{
```

Comment:
используйте file-scoped неймспейсы
```suggestion
namespace Tests;
```